### DISCUSSION TITLE
PNPM Workspace integration

### DISCUSSION THREAD

**Discussion by 50rayn** [f5/unovis#490] [Q&A] [answered]
Hello @rokotyan ,
was just looking around the project and wondering if there are any plans for integrating [pnpm-workspace](https://pnpm.io/workspaces) in the project. For sure will be needed to be done some changes in dependencies, import and maybe some build scripts.

**Comment by rokotyan** [comment]
@50rayn (cc @lee00678) Currently, we have no plans to integrate `pnpm`. What do you think are the advantages of it besides faster package installation times?

We certainly welcome such PRs, and I'm personally interested in trying pnpm (I know it's becoming quite popular, but I never had an opportunity to work with it). However, we need to keep in mind that we've spent quite a bit of time configuring our existing setup, ensuring there are no packages with high vulnerabilities even in our development dependencies. I don't know how difficult it's going to be with pnpm.

**Comment by 50rayn** [comment, marked as answer]
I think we can Mark as done, since it was done in #686 .
